### PR TITLE
Helm chart: Add the ability to use a pre existing peristent volume claim

### PR DESCRIPTION
**Pull request by jpflouret**
# Problem
I have an existing persistent volume and persistent volume claim with my PMS database. I want to use this pvc with the PMS helm chart.

# Solution
This PR uses a common helm pattern where the user can provide the name of a pre existing volume claim and the chart will configure the PMS container to use that claim instead of creating a volumeClaimTemplate.

### CODE REVIEW

File: charts/plex-media-server/values.yaml [hunk 1]
```diff
@@ -32,6 +32,10 @@ pms:
   # the volume size to provision for the PMS database
   configStorage: 2Gi
 
+  # Name of an existing `PersistentVolumeClaim` for the PMS database
+  # NOTE: When it's set configStorage and storageClassName are ignored
```
Comment (cilindrox) [inline charts/plex-media-server/values.yaml:36]: nit: wording.

```suggestion
  # NOTE: When set, 'configStorage' and 'storageClassName' are ignored.
```

File: charts/plex-media-server/templates/statefulset.yaml [hunk 2]
```diff
@@ -30,6 +30,11 @@ spec:
       priorityClassName: {{ .Values.priorityClassName | quote }}
       {{- end }}
       volumes:
+      {{- if .Values.pms.configExistingClaim }}
+      - name: pms-config
+        persistentVolumeClaim:
+          claimName: {{ tpl .Values.pms.configExistingClaim . }}
```
Comment (cilindrox) [inline charts/plex-media-server/templates/statefulset.yaml:36]: Do we need to template the string here, or can we use the value directly instead?

```suggestion
          claimName: {{ .Values.pms.configExistingClaim | quote }}
```

**Review by cilindrox — COMMENTED** [review]
LGTM - thanks @jpflouret - Could you bump the Chart.version a `minor` so we can release this?

**Comment by jpflouret** [comment]
Suggestions applied. I could squash the commit with the review suggestions if you prefer that.

**Review by cilindrox — APPROVED** [review]
LGTM 👍 thanks @jpflouret !

I'll squash and merge this for release

**Comment by jpflouret** [comment]
I suppose I need to sign the commit as per the branch rules. I'll ammed the commits with signature and force push to my branch.

**Comment by cilindrox** [comment]
@jpflouret appreciate it!

**Comment by jpflouret** [comment]
@cilindrox done!

**Pull request merged**